crates/test-utils/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "alloy-test-utils"
```

Comment:
not sure about the name, because these are not only used for testing, for example run some simulations using anvil.

perhaps alloy-nodes, alloy-node-bindings, alloy-node-commands|cmd

Reply:
https://github.com/alloy-rs/alloy/pull/111/commits/4d49fbe125e17c099da49688c547943fdd656b0c